### ISSUE TITLE
theme mismatch?  

### ISSUE THREAD

**Issue by isaackearl** [VemtoOrg/vemto2-issues#114]
I did a standard jetstream + filament site, and when I go to it I see the following:
it's like the top bar is dark mode and the rest is light.
I'm on 2.0.8 and I just started a fresh project didn't do any customization.
<img width="1305" alt="Screenshot 2024-05-06 at 12 48 54 AM" src="https://github.com/TiagoSilvaPereira/vemto2-issues/assets/9354830/896e56f1-83f0-4b8f-9e93-997e111100f3">

**Comment by TiagoSilvaPereira** [comment]
Hi, @isaackearl; I think it is more probable to be a Jetstream issue, as Vemto just installed it. Did you try running npm run dev to see if recompiling assets solves this problem?

**Comment by isaackearl** [comment]
> Hi, @isaackearl; I think it is more probable to be a Jetstream issue, as Vemto just installed it. Did you try running npm run dev to see if recompiling assets solves this problem?

Yes I did try that.  Also, those two drop downs on the top bar are definitely not added by default... Those are added by vemto, so yes it was just installed but it was also modified.

**Comment by TiagoSilvaPereira** [comment]
Ok, we'll check this. Maybe some recent change to jetstream causing incompatibilities with our navigation file

**Comment by nicollassilva** [comment]
@isaackearl You probably have a cache with the theme set to dark, and Vemto is generating a menu with dark classes, but the Jetstream layout does not have one, resulting in an inconsistency in the themes.

**Comment by TiagoSilvaPereira** [comment]
@isaackearl did you set Filament as Dark? Maybe this configuration is cached and causing this  issue

**Comment by vimitali368** [comment]
Maybe, the dark theme is related to the Windows theme settings. After selecting a light theme in Windows, the theme of the top menu of the web application has also changed.
https://github.com/TiagoSilvaPereira/vemto2-issues/issues/133#issuecomment-2108563521
